CalibTracker/SiStripHitEfficiency/src/HitEff.cc

```diff
@@ -408,7 +408,7 @@ void HitEff::analyze(const edm::Event& e, const edm::EventSetup& es){
 	  const TrajectoryStateOnSurface tsosTEC9 = itm->updatedState();
 	  
 	  // check if track on positive or negative z
-	  if (!iidd ==  StripSubdetector::TEC) cout << "there is a problem with TEC 9 extrapolation" << endl;
+	  if (!(iidd ==  StripSubdetector::TEC)) cout << "there is a problem with TEC 9 extrapolation" << endl;
```

Comment:
this appears to be the only "bare" cout (not protected by DEBUG). 
The bug disabled it.
If it should stay exposed for default processing, it has to be converted to LogWarning/LogInfo.
Otherwise, please add "& DEBUG"